addon/nodes/doc.ts

```diff
@@ -117,10 +105,12 @@ export const docWithConfig = ({
       },
     ],
     toDOM(node) {
+      console.log('rendering doc', node);
```

Comment:
console.log

Reply:
fixed in [1c528ec](https://github.com/lblod/ember-rdfa-editor/pull/1229/commits/1c528ec9dc93bbeee04271788d37ed3c9dffeab1)